packages/components/src/AboutDialog/AboutDialog.snapshot.test.js

```diff
@@ -23,24 +23,22 @@ describe('AboutDialog', () => {
 	});
 
 	it('should render', () => {
-		const wrapper = shallow(<AboutDialog.WrappedComponent {...props} />);
+		const wrapper = shallow(<AboutDialog {...props} />);
```

Comment:
There is no `WrappedComponent ` anymore as the i18n HOC mock returns the original component.